modulemd/katello_modulemd.yaml

```diff
@@ -0,0 +1,25 @@
+---
+document: modulemd
+version: 2
+data:
+    name: katello
+    stream: latest
+    version: 0
+    arch: x86_64
+    summary: Katello module
+    description: >-
+        A module to make Katello easier to install.
+    license:
+        module:
+            - GPLv3
+    profiles:
+        installer:
+            rpms:
+                - foreman-installer-katello
+    dependencies:
+      - buildrequires:
+            platform: ['el8']
+            foreman: ['latest']
+        requires:
+            foreman: ['latest']
```

Comment:
I guess versioning here would be nice since you can match the right Foreman to the Katello release, but given we don't have this today it's not a regression.